db/models/flags_v2/flag_update.py

```diff
@@ -16,8 +16,7 @@
 class FlagStatus(IntEnum):
     RAISED = 0
     STOPPED = 1
-    QA_COMPLETED = 2
-    RESOLVED = 3
+    RESOLVED = 2
```

Comment:
we need to be careful with `RESOLVED = 2` change. Assessment has to be updated to reflect this change

Reply:
ahh yes I wonder how this will affect existing records in the db?

Reply:
I think in the db, enum name is stored not the enum value so no effect of changing `RESOLVED = 2` to on existing records. But for removing `QA_COMPLETED` i think there will be an issue for existing records. I guess night shelter is just started, not sure if any application has come to QA complete stage

Reply:
I've kept RESOLVED as 3 for now, can changes this back to 2 in the future if it will have no implications with data already in the assessment_store db 👍🏾 I will also update assessment frontend to consider QA_complete as a separate workflow rather than a flag.